app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+import requests
+
+
+URL = "http://api.weatherapi.com/v1/current.json"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    key = os.environ.get("API_KEY")
+    city = "Paris"
```

Comment:
maybe these two variables can be as constants

Reply:
got it, thank you